com.unity.cinemachine/Samples~/3D Samples/SimpleCar/MixCamerasBasedOnSpeed.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Cinemachine;
+using UnityEngine;
+
+[RequireComponent(typeof(CinemachineMixingCamera))]
+public class MixCamerasBasedOnSpeed : MonoBehaviour
+{
+    public float MaxSpeed;
+    public Rigidbody RigidBodyOfCar;
```

Comment:
So, when I said to you that we should be conscious of making custom sample scripts as generic as possible, this is exactly the kind of thing I was talking about.  Why Car?  Because it's a car in _this_ scene.  You should step back and look at this script from a more usage-agnostic perspective.